model/registry/gen-ai.yaml

```diff
@@ -71,6 +71,26 @@ groups:
         brief: The number of tokens used in the LLM response (completion).
         examples: [180]
         tag: llm-generic-response
+      - id: usage.token_type
+        stability: experimental
+        type:
+          allow_custom_values: true
+          members:
+            - id: input
+              stability: experimental
+              value: "input"
+              brief: 'Input tokens (Embeddings)'
+            - id: prompt
```

Comment:
any reason to differentiate `input` and `prompt`. Operation name would contain information if it's embedding or chat

Reply:
OpenAI calls it 'input' tokens when referring to embedding. There is no 'prompt' or 'completion' in the embedding operation.